doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc

```diff
@@ -0,0 +1,34 @@
+[id='test-designer-create-test-template-dmn-based-proc']
+= Creating a test scenario template for DMN-based test scenarios
+
+Create a test scenario template for DMN-based scenarios by following the procedure below to validate your rules and data.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project for which you want to create the test scenario.
```

Comment:
Minor Suggestion - "for which" or "that ... for":

s/In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project for which you want to create the test scenario./In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project that you want to create the test scenario for.

Reply:
Changed as suggested.